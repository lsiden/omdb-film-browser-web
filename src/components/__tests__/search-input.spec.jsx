import React from 'react'
import TestRenderer from 'react-test-renderer';

import SearchInput from 'components/search-input'

const defaultProps = () => ({
  onCancelClick: () => {},
})

const createWrapper = () => TestRenderer.create(<SearchInput {...defaultProps()} />)

test('SearchInput', () => {
  expect(createWrapper()).toMatchSnapshot()
})
