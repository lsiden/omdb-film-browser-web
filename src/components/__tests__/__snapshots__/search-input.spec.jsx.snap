// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput SearchInput 1`] = `
<div
  style={
    Object {
      "position": "relative",
      "width": "100%",
    }
  }
>
  <span
    charSet="utf-8"
    dangerouslySetInnerHTML={
      Object {
        "__html": "⚲",
      }
    }
    style={
      Object {
        "color": "#888",
        "fontSize": "20pt",
        "left": "6pt",
        "position": "absolute",
        "top": "-4pt",
        "transform": "rotate(-45deg)",
      }
    }
  />
  <input
    onChange={[Function]}
    placeholder="placeholder"
    style={
      Object {
        "color": "black",
        "fontSize": "12pt",
        "height": "24pt",
        "textIndent": "24pt",
        "width": "100%",
      }
    }
    type="search"
    value="value"
  />
  <span
    aria-label="Cancel"
    charSet="utf-8"
    className="hover-pointer"
    dangerouslySetInnerHTML={
      Object {
        "__html": "❌",
      }
    }
    onClick={[Function]}
    onKeyPress={[Function]}
    role="button"
    style={
      Object {
        "color": "#888",
        "fontSize": "8pt",
        "position": "absolute",
        "right": 6,
        "top": 6,
      }
    }
    tabIndex={0}
  />
</div>
`;
